src/collection-browser.ts

```diff
@@ -262,6 +262,12 @@ export class CollectionBrowser
   @property({ type: Object }) dataSource: CollectionBrowserDataSourceInterface =
     new CollectionBrowserDataSource(this, this.pageSize);
 
+  /**
+   * The maximum number of pages we will load when a privileged user clicks
+   * the "Manage" button on the search page. Limited to 15 pages.
+   */
+  maxPagesToManage = 15;
```

Comment:
Q: Could we make this a `@property`?

Reply:
I thought about it but after that it won't be accessible in CollectionBrowserDataSource.ts

Reply:
Yes, we can!